.github/workflows/build-cross-action.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           - 3.10.15
           - 3.11.10
           - 3.12.5
```

Comment:
3.12.7 is the most current, should this be that ?

Reply:
No, only adding 3.13.0 support in this PR. Upgrading others can happen but in a different PR.